### PR TITLE
MBM-287-288 Implemented telekinetic utility tiers 3 and 4

### PR DESCRIPTION
**Pull request by MattKirchh**
Had a late-night lightbulb go off, and banged this out in 30 minutes.

### CODE REVIEW

**Comment by MattKirchh** [comment]
Small problem, this doesn't work if the player has a chestplate that isn't from our mod or isn't wearing a chestplate at all.

**Comment by insertStringHere** [comment]
> Small problem, this doesn't work if the player has a chestplate that isn't from our mod or isn't wearing a chestplate at all. We may not be able to work around this.

we can make a bugfix to work on it later. For now, it's in the tooltip so it should be fine

**Review by insertStringHere — APPROVED** [review]

File: src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java [hunk 1]
```diff
@@ -40,24 +42,33 @@ public TelekineticUtilitySpell(Properties properties, TelekineticUtilitySpellCon
     public static void onPlayerTick(TickEvent.PlayerTickEvent event) {
         if (event.side.isServer() && event.phase == TickEvent.Phase.START && !event.player.isCreative()) {
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java:43]: we also gotta check for spectator here; I tried to use spectator mode and realized I couldn't fly lol

Comment (MattKirchh) [reply src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java]: Ha, fixed.

**Review by solidwaterslayer — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]

**Pull request merged**